compaction/time_window_compaction_strategy.cc

```diff
@@ -26,9 +26,9 @@ time_window_compaction_strategy_state& time_window_compaction_strategy::get_stat
     return table_s.get_compaction_strategy_state().get<time_window_compaction_strategy_state>();
 }
 
-time_window_compaction_strategy_options::time_window_compaction_strategy_options(const std::map<sstring, sstring>& options) {
+std::chrono::seconds time_window_compaction_strategy_options::validate_compaction_window_unit(const std::map<sstring, sstring>& options) {
+    const std::unordered_map<sstring, std::chrono::seconds> valid_window_units = { { "MINUTES", 60s }, { "HOURS", 3600s }, { "DAYS", 86400s } };
     std::chrono::seconds window_unit = DEFAULT_COMPACTION_WINDOW_UNIT;
-    int window_size = DEFAULT_COMPACTION_WINDOW_SIZE;
```

Comment:
This conflicts with https://github.com/scylladb/scylladb/pull/14043/commits/f5db89f2786bdc9a5862ac63b9a4ec8cf84cc5cd

Can you please base your series on top of the following patches?
https://github.com/scylladb/scylladb/pull/14043/commits/72001a7c413013bb78aa508e964cb5f3dfb1efc1
and
https://github.com/scylladb/scylladb/pull/14043/commits/f5db89f2786bdc9a5862ac63b9a4ec8cf84cc5cd